assets/src/design-system/components/contextMenu/animatedContextMenu.js

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useEffect, useLayoutEffect, useRef, useState } from 'react';
+import styled, { css } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { BEZIER } from '../../../animation';
+import { CORNER_DIRECTIONS, DIRECTIONS } from '../../utils/directions';
+import Menu, { MenuPropTypes } from './menu';
+import { Popover, Shadow } from './styled';
+
+const PERCENTAGE_OFFSET = {
+  [DIRECTIONS.TOP]: 100,
+  [DIRECTIONS.RIGHT]: 0,
+  [DIRECTIONS.BOTTOM]: -100,
+  [DIRECTIONS.LEFT]: -50,
+};
+
+const transition = `transform 0.175s ${BEZIER.outSine}`;
+const initialScale = 0.5;
+const fullSize = css`
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+`;
+
+const MenuWrapper = styled.div`
+  position: absolute;
+`;
+
+const MenuRevealer = styled.div`
+  overflow: hidden;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  ${fullSize}
+`;
+
+const MenuCounterRevealer = styled.div`
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+`;
```

Comment:
updated border radius from 8px to `theme.borders.radius.small` (`4px`). This matches the new designs.